helm/cluster-eks/values.yaml

```diff
@@ -13,8 +13,8 @@ cluster:
           group: controlplane.cluster.x-k8s.io
           kind: AWSManagedControlPlane
           version: v1beta2
-        infrastructureMachineTemplateSpecTemplateName: not-used
-    provider: aks
+        infrastructureMachineTemplateSpecTemplateName: template-not-implemented
```

Comment:
even though the control plane rendering is disabled via the `resourcesApi` values, a template name must be supplied here. The named template does nothing, it just exists to satisfy the restrictions.

Reply:
see https://github.com/giantswarm/cluster-eks/pull/136/files#diff-0480e12875bac9219670d1d1844f3ca6af41e3e63521610e7bdb801a2659bbeeR75